QQE-1584.md

```diff
@@ -0,0 +1,82 @@
+# QQE-1584 Quarkus LangChain4J testing
+
+JIRA link: https://issues.redhat.com/browse/QQE-1584
+
+This is a high-level strategy for testing of langchain4j
+
+## Scope of the testing
+@SystemMessage + @UserMessage annotations
+RAG functionality
+Integration with MCP API
+
+### Optional, but desirable:
+@Tool + @RegisterAiService (only supported for OpenAI, so cannot be tested locally)
+Observability for tool requests (to avoid a situation, where LLM overwhelms the database with too many requests)
+
+## Existing coverage upstream:
+### Workshop
+https://quarkus.io/quarkus-workshop-langchain4j/step-11/#running-the-llm-inference-locally  and  https://github.com/quarkusio/quarkus-workshop-langchain4j/tree/main/step-11
+(they are not run automatically, as far as I can tell)
```

Comment:
Is this something that would be helpful to have for us to be able to verify tech preview support for langchain4j? If so, it should be part of strategy.